src/components/Menu/index.tsx

```diff
@@ -24,6 +25,8 @@ const Menu = (props) => {
   const activeMenuItem = getActiveMenuItem({ menuConfig: config(t), pathname })
```

Comment:
here can lift it to useMenuConfig or something

Reply:
Done